development/probes/probe-k8s-metrics-server/src/main/java/eubr/atmosphere/tma/probes/ProbeKubernetes.java

```diff
@@ -169,13 +175,10 @@ private static void parsePodMetrics(InputStreamReader isr, BackgroundClient clie
 
                     String memoryString = ltmUsage.get("memory").toString();
                     message.addData(parseDatumValue(memoryString, memoryDescriptionId, 2));
-
-                    System.out.println(cpuString);
-                    System.out.println(memoryString);
                 }
                 message.setSentTime(Calendar.getInstance().getTimeInMillis());
                 message.setMessageId(messageId++);
-                System.out.println(message);
+                LOGGER.info(message.toString());
```

Comment:
Avoid calling toString() inside LOG, because it will be evaluated even if the LOG is ignored.
(see multiple instances)